src/containers/Pages/CoursePage.js

```diff
@@ -0,0 +1,147 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import { withRouter } from "react-router-dom";
+import {
+  getTutorialsByCourseId,
+  getQuizesByCourseId,
+  getCourseById
+} from "../../store/actions/Actions";
+import Footer from "../../components/Footer";
+import "./CoursePage.scss";
+
+class CoursePage extends Component {
+  getTutorialId = (tutorialId, name) => {
+    this.props.history.push(`/tutorial-page/${name}/${tutorialId}`);
+  };
+
+  getQuizId = quizId => {
+    this.props.getQuizes(quizId);
+  };
+
+  componentDidMount() {
+    const pathname = this.props.history.location.pathname;
+    var id = pathname.split("/");
+    this.props.getCourses(id[3]);
+    this.props.getTutorials(id[3]);
+    this.props.getQuizes(id[3]);
+  }
+
+  render() {
+    let tutorials, quizes;
+    if (!this.props.tutorials) {
+      tutorials = (
+        <div className="TutorialShow">
+          <p>There is no tutorial available</p>
+        </div>
+      );
+    } else {
+      tutorials =
+        this.props.tutorials &&
+        this.props.tutorials.map((tutorial, index) => (
+          <div
+            className="tutorial-name"
+            key={index}
+            onClick={() =>
+              this.getTutorialId(tutorial.tutorialId, tutorial.name)
+            }
+          >
+            <ul>
+              <li> {tutorial.name}</li>
+            </ul>
+          </div>
+        ));
+    }
+    if (!this.props.quizes) {
+      quizes = (
+        <div className="quizShow">
+          <p>There is no quiz available</p>
+        </div>
+      );
+    } else {
+      quizes =
+        this.props.quizes &&
+        this.props.quizes.map((quiz, index) => (
+          <div
+            className="tutorial-name"
+            key={index}
+            onClick={() => this.getQuizId(quiz.quizId)}
```

Comment:
@Humnaa 
why is this method named `getQuizId`?

Reply:
I have renamed it getQuizById